recycler.sh

```diff
@@ -254,6 +254,21 @@ recreate_volume() {
   return 0
 }
 
+# Verify that a volume is not mounted anywhere. Returns the number of fuse
+# clients that are mounting the specified volume.
+#
+# $1 - list of gluster endpoints
+# $2 - name of the volume
+volume_unused() {
+  local rhost="${1%%,*}"  # extract the first entry
+  local vol_name="$2"
+  nfuse="$(gluster --remote-host="$rhost" volume status "$vol_name" client-list | \
+    awk '/^\w+\s+[[:digit:]]+$/{print $2}')"
```

Comment:
Don't this also match the last line with the summary? How does this handle the self-heal daemon?

Reply:
Good catch, copy/paste fail